src/app/base/components/record-card/record-card.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { DataBrowserModals, RecordCommentsModal } from 'src/app/data/components/data-browser/helper-classes/modals-helper';
+
+
+@Component({
+  selector: 'kern-record-card',
+  templateUrl: './record-card.component.html',
+  styleUrls: ['./record-card.component.scss']
+})
+export class RecordCardComponent implements OnInit {
+
+  @Input() record: Record;
+  @Input() attributesSortOrder: AttributeSort;
+  @Input() index: number;
+  @Input() extendedRecords: ExtendedRecord;
```

Comment:
this should be singular, right? we only display one record per card

Reply:
ok i might have misread something here. Why is the extendedRecord needed? (the structure to hold all records in the record specific component)

Reply:
I see that the session ID is used somewhere in the HTML -> we can of course provide that. 


So in general a bit unclear on the direction we want to go with this component.
This could be A: To reuse the component (e.g. in the display for the "View" for run on 10 for heuristics) then we need to generalize a bit more (like record & attribute arrays and some optional stuff) 
Or B: Use this only as a code restructure tool only meant for the databrowser -> then we should organize it below the data browser in the structure of the folders


-> I think I would go with A to further reduce the complexity where we can -> but we can talk about this